src/crimson/net/SocketMessenger.cc

```diff
@@ -165,9 +165,30 @@ SocketMessenger::bind(const entity_addrvec_t& addrs)
                           [this, addrs] (auto& count) {
     return seastar::repeat_until_value([this, addrs, &count] {
       assert(count >= 0);
+
+      int ms_bind_port_min=0;
+      int ms_bind_port_max=0;
+
+      if (this->get_myname().is_osd() &&
+          local_conf()->ms_bind_port_min_osd &&
+          local_conf()->ms_bind_port_max_osd) {
+        ms_bind_port_min = local_conf()->ms_bind_port_min_osd;
+        ms_bind_port_max = local_conf()->ms_bind_port_max_osd;
+      }
+      else if (this->get_myname().is_mds() &&
```

Comment:
nit: let's keep the parenthesis at the same line as `else` (and `else if` as well). `} else {` is the convention.

This comment applies to other places in the code too.

Reply:
> Also, I'm not sure the PR matches the referenced ticket (https://tracker.ceph.com/issues/54978).

It seems to have been copied incorrectly while referring to the form of other commits. (shame)